Assets/Scripts/Terrain/HeightMap/ChangeMap.cs

```diff
@@ -0,0 +1,86 @@
+
+/// <summary>
+/// A height map that is meant to hold changes. Starts will all avlues
+/// initialized to zero and can be applied to other height maps.
+/// </summary>
+public class ChangeMap : HeightMap {
+    /// <summary>
+    /// Data stored in change map
+    /// </summary>
+    private float[] map;
+    /// <summary>
+    /// Dimensions of the map in the x and y axis
+    /// </summary>
+    private int sizeX, sizeY;
```

Comment:
perhaps dimX and dimY, makes them more identifiable at the dimensions

Reply:
Yes, those are better names. I have updated them